doc/source_interpreter.tex

```diff
@@ -11,4 +9,139 @@ \section*{Interpreter Support}
+&&                       && |   &\quad && \textbf{\texttt{function}}\  \textit{name} \ 
+                                   \textbf{\texttt{(}}\  \textit{parameters} \ \textbf{\texttt{)}}\ \textit{block} \quad
+                                                           &&  \textrm{treat as}:\ \textbf{\texttt{const}}\  \textit{name} \ 
+                                   \textbf{\texttt{=}}\  \textit{parameters} \ \textbf{\texttt{=>}}\ \textit{block} \textbf{\texttt{;}} \\
+&&                       && |   &\quad && \textbf{\texttt{return}}\  \textit{expression} \ \textbf{\texttt{;}}
+                                                           && \texttt{list("return\_statement", <expression>)} \\
+&&                       && |   &\quad && \textit{if-statement} \quad
+                                                           && \textrm{see below}\\
+&&                       && |   &\quad && \textbf{\texttt{while}}\  
+                                   \textbf{\texttt{(}}\  \textit{expression} \ \textbf{\texttt{)}} \
+                                   \textit{block}
+                                                           && \texttt{list("while\_loop", <expression>, <statement>)} \\
+&&                       && |   &\quad && \textbf{\texttt{for}}\ \textbf{\texttt{(}} \ 
+                                          (\ \textit{\hyperlink{for}{assignment}} \ | \  \textit{\hyperlink{for2}{let}}\ ) \textbf{\texttt{;}} \\
+&&                       &&     &\quad && \ \ \ \ \ \ \ \ \ \ \textit{expression} \ \textbf{\texttt{;}} \\
+&&                       &&     &\quad && \ \ \ \ \ \ \ \ \ \ \textit{assignment} \ \textbf{\texttt{)}} \ 
+                                            \textit{block}
+                                            && \texttt{list("for\_loop", <statement>, <expression>, <statement>,}\\
+                                            &&&&&&&&&\ \ \ \ \ \ \texttt{ <statement>)} \\
+&&                       && |   &\quad && \textbf{\texttt{break}}\ \textbf{\texttt{;}}
+                                                           && \texttt{list("break\_statement")} \\
+&&                       && |   &\quad && \textbf{\texttt{continue}}\ \textbf{\texttt{;}}
+                                                           && \texttt{list("continue\_statement")} \\
+&&                       && |   &\quad &&  \textit{block} 
+                                                           && \textrm{see below}\\
+&&                       && |   &\quad &&  \textit{expression} \ \textbf{\texttt{;}}
+                                                           && \textrm{see below}\\[1mm]
+&& \textit{parameters}   && ::= &\quad &&  \epsilon\ | \  \textit{name} \ 
+                                                   (\ \textbf{\texttt{,}} \ \textit{name}\ )\ \ldots
+                                                            && \texttt{list of strings} \\
```

Comment:
Should be list of name.

Also not actually used, unless you want the function definition specs to refer to this instead of writing "list of name" directly.